### PR TITLE
https://issues.redhat.com/browse/ACM-8999 12/13/23 ACM 2.9.1 Errata

### PR DESCRIPTION
**Pull request by dockerymick**
https://issues.redhat.com/browse/ACM-8999

### CODE REVIEW

File: release_notes/errata.adoc [hunk 1]
```diff
@@ -7,3 +7,12 @@ By default, Errata updates are automatically applied when released. The details
 
 See link:../install/upgrade_hub.adoc#upgrading-by-using-the-operator[Upgrading by using the operator] for more information about upgrades.
 
+== Errata 2.9.1
+
+* Delivers updates to one or more product container images. 
+
+* Fixes an issue that caused policies with empty `key` values to not applied. (link:https://issues.redhat.com/browse/ACM-7810[ACM-7810])
```
Comment (cadawson) [inline release_notes/errata.adoc]:
```suggestion
* Fixes an issue that caused policies with empty `key` values to not be applied. (link:https://issues.redhat.com/browse/ACM-7810[ACM-7810])
```

**Review by cadawson — CHANGES_REQUESTED** [review]
Quick minor change, otherwise looks great! Thanks!

**Comment by dockerymick** [comment]
@cadawson thanks Chris, I committed your suggestion

**Review by cadawson — APPROVED** [review]

**Comment by dockerymick** [comment]
@cadawson thanks Chris, merging this

**Pull request merged**